animations/effects.json

```diff
@@ -286,9 +335,8 @@
 	],
 	"effect:spell-effect-shields-of-the-spirit-security": [
 		{
-			"trigger": "effect",
 			"preset": "onToken",
-			"file": "jb2a.markers.shield_rampart.complete.01.orange",
+			"file": "jb2a.markers.shield_rampart.loop.03.blue",
```

Comment:
Why the loop and not the complete version?

Reply:
uhh just grabbed wrong one mb

Reply:
fixed